src/components/community/Platforms/platformsData.ts

```diff
@@ -0,0 +1,34 @@
+import { DiscordLogo, TelegramLogo } from "phosphor-react"
+
```

Comment:
rename to platformsContent and update the data to match the current state of Figma

Reply:
The success state is now also different for the platforms, should we create different components, that we put in platformsContent, and render from there? (like the logos)
We should discuss, what the format of these states will be, but for the demo, I'm just going to hardcode the new discord success state, since that's the only one we see now.